#! /bin/sh -e

if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
elif [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch $pdir -f --no-backup-if-mismatch -p1 < $0;;
    -unpatch) patch $pdir -f --no-backup-if-mismatch -R -p1 < $0;;
    *)
	echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
	exit 1
esac
exit 0

# DP: send bug reports to Debian bash maintainer too.

--- ./support/bashbug.sh~	Mon Nov 27 17:15:21 2000
+++ ./support/bashbug.sh	Tue Feb  6 18:54:48 2001
@@ -98,6 +98,7 @@
 		esac ;;
 esac
 
+BUGBASH="${BUGBASH},bash@packages.debian.org"
 BUGADDR="${1-$BUGBASH}"
 
 if [ -z "$DEFEDITOR" ] && [ -z "$EDITOR" ]; then
